packages/grafana-ui/src/components/ValuePicker/ValuePicker.story.tsx

```diff
@@ -35,6 +35,12 @@ export default {
         options: getAvailableIcons(),
       },
     },
+    size: {
+      control: {
+        type: 'select',
+        options: ['sm', 'md', 'lg'],
+      },
```

Comment:
```suggestion
    size: {
      options: ['sm', 'md', 'lg'],
      control: {
        type: 'select',
      },
```
Options as part of controls are going to be deprecated, instead they'd be directly on the args. Can you do the same for the other options in this story? Also `knobs.disabled` should be `knobs.disable` instead.

Reply:
okidoki 👍